py_modules/mail_api.py

```diff
@@ -65,28 +67,26 @@ def get_mails(self, step):
                     "bodyPLAIN": str(obj['bodyPLAIN']),
                     "attach": obj['attach'],
                     "directory": str(mail[4]),
-                    "datetimes": str(strdate),
+                    "datetimes": str(parsed_date),
                     "readed": str(mail[5]),
                 }
 
-                dict_mails.append(tempmail)
+                dict_mails.append(temp_mail)
             except Exception as e:
```

Comment:
Se riusciamo a essere più precisi con l'eccezione da catturare, ci verrà più semplice debuggare nel caso in cui avremo uno scenario in cui viene catturata

Reply:
Non avendo sviluppato quella parte non saprei che tipo di eccezione aspettarmi... @MattiaFailla @basic-ph @nocturn9x voi avete qualche idea?

Reply:
Su questa eccezione sto investigando. Sembra che cambi molto tra vari sistemi operativi e interfacce di rete.

Reply:
Aperto issue https://github.com/MattiaFailla/ToooMail/issues/59

Reply:
Lo vogliamo risolvere in questo branch? O lo lavoriamo separatamente?

Reply:
Separatamente.. è un bug non bloccante. Così mergiamo questo e risolviamo direttamente in dev o su branch dedicato.

Reply:
Allora chiudo la conversazione.